docs/tutorials/elasticdl_on_prem_cluster.md

```diff
@@ -110,3 +111,45 @@ class MyCluster:
 
 cluster = MyCluster()
 ```
+
+Then, we can use the `spec.py` to submit jobs.
+
+Firstly, we need to use `spec.py` to initialize model zoo
+
+```bash
+elasticdl zoo init --cluster_spec=spec.py
+```
+
+After building images using above commands, we need to use `spec.py` to
```

Comment:
`above commands` only init the zoo. In order to make it clear, we can rewrite above commands here.